src/install/kubernetes/helm/install-helm.mdx

```diff
@@ -0,0 +1,121 @@
+        * `newrelic-prometheus-agent.enabled=true`: Will deploy our Prometheus Agent, which automatically collects data from Prometheus endpoints present in the cluster.
+        * `nri-metadata-injection.enabled=true`: Will install our minimal webhook, which adds environment variables that, in turn, allows [linking applications instrumented with New Relic APM to Kubernetes](/docs/kubernetes-pixie/kubernetes-integration/link-your-applications/link-your-applications-kubernetes).
+
+    Our Kubernetes charts have a comprehensive set of flags and tunables that can be edited to better fit your particular needs. Please, check the [Configure the integration](#configure) section below to see what can be changed.
+
+5. Install the Kubernetes integration by running the command without `--debug` and `--dry-run`:
+
+    ```shell
+    helm upgrade --install newrelic-bundle newrelic/nri-bundle \
+    --namespace newrelic --create-namespace \
+    -f values-newrelic.yaml
+    ```
+    
+6. Run this command to check that pods are deploying and reaching a stable state:
+
+    ```shell
+    kubectl -n newrelic get pods -w
+    ```
+
+7.  If you're going to use a GKE AutoPilot cluster, your Helm command should include the following:
+
+    ```yaml
+    --set newrelic-infrastructure.controlPlane.enabled=false \
+    --set newrelic-infrastructure.kubelet.config.scheme=http \
+    --set newrelic-infrastructure.kubelet.config.port=10255 \
+    ```
+
+    [Fluent Bit](https://fluentbit.io/) can use a FileStore volume to prevent data loss or duplicated logs during Fluent Bit pod restarts or redeploys.  Google FileStore API needs to be enabled in your GKE project. Check this [document](https://support.google.com/googleapi/answer/6158841) to know more about enabling APIs. 
+
+    To use FileStore, add the following:
+
+    ```yaml
+    --set newrelic-logging.fluentBit.persistence.mode=persistentVolume
+    --set newrelic-logging.fluentBit.persistence.persistentVolume.storageClass=standard-rwx
+    ```
+
+    <Callout variant="important">
+    The use of FileStore will result in additional costs, as charged by Google. See your Google Cloud Admin or the [FileStore](https://cloud.google.com/filestore/pricing) doc for more info.
+    </Callout>
+
+    If you don't want to use FileStore, add the following:
+
+    * Set `"newrelic-logging.fluentBit.persistence.mode"` to `"none"`
+    * Remove `"newrelic-logging.fluentBit.persistence.persistentVolume.storageClass"`
+
+8. Decide if you want to install and configure the Prometheus endpoints to scrape. See [Install and configure the Prometheus agent on Kubernetes clusters](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/) for more info. We recommend to scrape all Prometheus endpoints except core Kubernetes system metrics. Kubernetes integration already collects core Kubernetes system metrics in other components. 
```

Comment:
I would remove points 8 and 9, we already talk about Prometheus and Logging in other documents and in the "Reduce Data Ingest" doc.